reference/commands/graph/info.rst

```diff
@@ -19,7 +19,7 @@ conan graph info
                             [--package-filter PACKAGE_FILTER] [--deploy DEPLOY]
                             [path]
 
-    Compute the dependency graph and shows information about it.
+    Compute the dependency graph and show information about it.
```

Comment:
This is a super-minor detail but I think the output of the latest release is shows not show, this should be also changed in the Conan Client. We can merge this and fix in the Conan Client for 2.0.2